oauth2/oauth2-client/src/test/java/org/springframework/security/oauth2/client/oidc/authentication/OidcAuthorizationCodeAuthenticationProviderTests.java

```diff
@@ -302,6 +326,37 @@ public void authenticateWhenTokenSuccessResponseThenAdditionalParametersAddedToU
 				this.accessTokenResponse.getAdditionalParameters());
 	}
 
+	// gh-4442
+	@Test
+	public void authenticateWhenTokenSuccessResponseThenAdditionalParametersAddedToUserRequestNoNonce() {
```

Comment:
I'm not sure I understand the purpose for this test. I think we can remove this?